pkg/reconciler/taskrun/resources/taskref_test.go

```diff
@@ -289,43 +280,6 @@ func TestGetTaskFunc(t *testing.T) {
 		expectedKind v1beta1.TaskKind
 	}{
 		{
-			name:       "remote-task",
```

Comment:
I think it would make sense to create a new test class for testing "bundle" syntax. We already have tests for local taskrefs and remote resolution, and this test class mixes tests for local taskrefs with tests for bundle syntax. The new bundle syntax test class can be removed when support for the bundle syntax is removed in the v1 PR.

Reply:
updated, thanks @lbernick  for pointing out and being thorough!